.eslintcache

```diff
@@ -1 +1 @@
results":"11","hashOfConfig":"12"},{"size":502,"mtime":1722632906938,"results":"13","hashOfConfig":"12"},{"size":1915,"mtime":1722668030509,"results":"14","hashOfConfig":"12"},{"size":1835,"mtime":1722633025876,"results":"15","hashOfConfig":"12"},{"size":1877,"mtime":1722667757514,"results":"16","hashOfConfig":"12"},{"size":824,"mtime":1722667997491,"results":"17","hashOfConfig":"12"},{"size":724,"mtime":1722668010007,"results":"18","hashOfConfig":"12"},{"size":5629,"mtime":1722668026551,"results":"19","hashOfConfig":"12"},{"size":166,"mtime":1722632836064,"results":"20","hashOfConfig":"12"},{"size":2179,"mtime":1722632835999,"results":"21","hashOfConfig":"12"},{"filePath":"22","messages":"23","suppressedMessages":"24","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"4hgr45",{"filePath":"25","messages":"26","suppressedMessages":"27","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"28","messages":"29","suppressedMessages":"30","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"31","messages":"32","suppressedMessages":"33","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"34","messages":"35","suppressedMessages":"36","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"37","messages":"38","suppressedMessages":"39","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"40","messages":"41","suppressedMessages":"42","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"43","messages":"44","suppressedMessages":"45","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"46","messages":"47","suppressedMessages":"48","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"49","messages":"50","suppressedMessages":"51","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/alial-kheder/Desktop/nextapp/nextapp/test.js",[],[],"/Users/alial-kheder/Desktop/nextapp/nextapp/src/app/layout.tsx",[],[],"/Users/alial-kheder/Desktop/nextapp/nextapp/src/app/page.tsx",[],[],"/Users/alial-kheder/Desktop/nextapp/nextapp/src/components/ui/button.tsx",[],[],"/Users/alial-kheder/Desktop/nextapp/nextapp/src/components/ui/card.tsx",[],[],"/Users/alial-kheder/Desktop/nextapp/nextapp/src/components/ui/input.tsx",[],[],"/Users/alial-kheder/Desktop/nextapp/nextapp/src/components/ui/label.tsx",[],[],"/Users/alial-kheder/Desktop/nextapp/nextapp/src/components/ui/select.tsx",[],[],"/Users/alial-kheder/Desktop/nextapp/nextapp/src/lib/utils.ts",[],[],"/Users/alial-kheder/Desktop/nextapp/nextapp/tailwind.config.ts",[],[]]
```

Comment:
This file should not be pushed to the git repo, 
- you should add it in the `.gitignore` file
- you should remove it from the repo